.travis.yml

```diff
@@ -16,24 +25,19 @@ install:
   - 'echo "Apache Ant ARCHIVE: $APACHE_ANT_BASE"'
   - '[ "${TRAVIS_OS_NAME}" = "linux" ] && wget http://apache.mirror.iphh.net/ant/binaries/$APACHE_ANT_BASE-bin.tar.gz && tar xzf $APACHE_ANT_BASE-bin.tar.gz && sudo mv $APACHE_ANT_BASE /usr/local/$APACHE_ANT_BASE && sudo rm -f /usr/local/ant && sudo ln -s /usr/local/$APACHE_ANT_BASE /usr/local/ant && sudo ln -s /usr/local/$APACHE_ANT_BASE/bin/ant /usr/local/bin/ant || true'
   - '[ "${TRAVIS_OS_NAME}" = "linux" ] && sudo apt-get -y install texinfo || true'
-  - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew update || true'
-  - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew uninstall libtool && brew install libtool || true'
-  - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew install ant || true'
+  # - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew update || true'
+  # - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew uninstall libtool && brew install libtool || true'
+  # - '[ "${TRAVIS_OS_NAME}" = "osx" ] && brew install ant || true'
 
 script:
-  - if [[ "${TRAVIS_CPU_ARCH}" != "arm64" && "x$JDK" != "xdefault" ]]; then wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh && export JAVA_HOME=$(bash install-jdk.sh --feature $JDK --emit-java-home | tail -1); export PATH=$JAVA_HOME/bin:$PATH; fi || true
+  # - if [[ "${TRAVIS_CPU_ARCH}" != "arm64" && "x$JDK" != "xdefault" ]]; then wget https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh && export JAVA_HOME=$(bash install-jdk.sh --feature $JDK --emit-java-home | tail -1); export PATH=$JAVA_HOME/bin:$PATH; fi || true
   - ant test
   - ant test-platform
-  - if [ "x$JDK" == "xdefault" ]; then ant checkstyle; fi
-  - if [ "x$JDK" == "xdefault" ]; then ant dist; fi
+  # - if [ "x$JDK" == "xdefault" ]; then ant checkstyle; fi
+  # - if [ "x$JDK" == "xdefault" ]; then ant dist; fi
 
 jobs:
-    include:
-        - env: JDK=default
-        - env: JDK=11
-        - env: JDK=ea
-        - env: JDK=11
-          arch: arm64
-        - env: JDK=11
-          os: osx
- 
+  include:
+    - name: "Linux JDK11 ARM"
+      arch: arm64
+      # env: JDK=11

```

Comment:
If memory serves, we had to add specific JDK because the default Travis JDK changed. Travis had a nasty habit of default JDK's changing, and causing build failures that were not obvious due to the "silent" default JDK change. Happy to remove the specific JDK config, but wanted to mention it in case oddities reoccur in the future, maybe someone else will remember to look for a default JDK change.

Reply:
I think this particular one is only relevant in the `script` section above as an input to the `install-jdk` script, so is necessary for other matrix choices.  For the ARM-only test, we load our own JDK in the `install:` section.  This would only be necessary if JNA's `ant` files need this environment variable set.  So this falls in the same category of "leave it there commented out as a prompt in case we need to put it back".